src/test/java/uk/gov/hmcts/cmc/claimstore/models/serialization/CountyCourtJudgmentSerializationTest.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.hmcts.cmc.claimstore.models.serialization;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.InvalidTypeIdException;
+import org.junit.Test;
+import uk.gov.hmcts.cmc.claimstore.config.JacksonConfiguration;
+import uk.gov.hmcts.cmc.claimstore.models.CountyCourtJudgment;
+import uk.gov.hmcts.cmc.claimstore.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.claimstore.models.sampledata.SampleCountyCourtJudgment;
+import uk.gov.hmcts.cmc.claimstore.models.sampledata.SampleRepaymentPlan;
+import uk.gov.hmcts.cmc.claimstore.models.sampledata.SampleTheirDetails;
+import uk.gov.hmcts.cmc.claimstore.utils.ResourceReader;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CountyCourtJudgmentSerializationTest {
+
+    private static final ObjectMapper mapper = new JacksonConfiguration().objectMapper();
+
+    @Test
+    public void shouldConvertCCJJsonToJavaForIndividual() throws IOException {
+
+        //given
+        TheirDetails defendant = SampleTheirDetails.builder().withRepresentative(null).individualDetails();
+        CountyCourtJudgment expected = SampleCountyCourtJudgment.builder().withDefendant(defendant).build();
+
+        //when
+        CountyCourtJudgment other = jsonToModel("/county-court-judgment/individual-defendant.json");
```

Comment:
I think there's too much fixture testing here,
Its not maintainable and can't be refactored.

what do you think?

Reply:
Well, I think we should test mapping somewhere. I agree that this might require a lot of work, but we need to do it somehow. If we don't have this tests we should always test all options manually - which is boring and time-consuming. 

I'd rather leave it here. What's more, I believe we should add this kind of tests for other models.

Reply:
you're right we need to test it but I don't think we should be testing party type serialisation in CCJ, it shouldn't be relevant. It should be covered as part of PartyType tests?
@rlewan your thoughts too?

Reply:
If we want to add more tests then I'm happy to remove tests for these cases from my code. 

I can add tests for it (PartyType) and then remove theses. What do you think?

Reply:
I would prefer that, even happier if you do it in a different PR.